README.md

```diff
@@ -600,13 +600,15 @@ _See also: [IT Asset Management]([Ralph](#it-asset-management))_
 *Network configuration management tools.*
 
 - [GNS3](https://www.gns3.com/) - Graphical network simulator that provides a variety of virtual appliances - ([Source Code](https://github.com/GNS3/gns3-gui/), [Documentation](https://docs.gns3.com/)) - `GPLv3`
+- [OpenWISP](https://openwisp.org/) - Open Source Network Management System for [OpenWRT](https://openwrt.org/) based routers and access points. ([Demo](https://openwisp.org/demo.html), [Source Code](https://github.com/openwisp)) `GPLv3` `Python` 
```

Comment:
```suggestion
- [OpenWISP](https://openwisp.org/) - Open Source Network Management System for OpenWRT based routers and access points. ([Demo](https://openwisp.org/demo.html), [Source Code](https://github.com/openwisp)) `GPL-3.0` `Python` 
```

Reply:
I think the official name is GPLv3 and not GPL-3.0 see https://www.gnu.org/licenses/gpl-3.0.html

Reply:
I am using it since a few weeks in a professional setup and I am very happy and looking into contributing myself as I want to make some features that I need. 
Currently I have only 5 routers (PC engines apu boards) in the system, but have ~400 devices running in my own system that I will migrate over time into OpenWISP.a
I accidentally found OpenWISP when I was looking into a new OS for my routers (OpenWRT) and I am very impressed by the feature-rich, well-build platform that can easily be extended.

Reply:
> I think the official name is GPLv3 and not GPL-3.0 see https://www.gnu.org/licenses/gpl-3.0.html

We use [SPDX identifiers](https://spdx.org/licenses/). I will commit the fix.

Thank you for providing additional info, I think we should merge this. It's also helpful for defining possible criteria for inclusion being discussed in https://github.com/awesome-foss/awesome-sysadmin/pull/357.